qemu/tests/expose_host_mtu.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Expose host MTU to guest test
+
+    1) Boot up guest with param 'host_mtu=4000' in nic part
+    2) Disable NetworkManager in guest
+    3) set mtu of guest tap (eg: tap0) and physical nic (eg: eno1) to 4000 in host
+    4) check the mtu in guest
+    5) ping from guest to external host with packet size 3972
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    host_interface = None
+    if params["netdst"]:
+        host_interface = params["netdst"]
+        if host_interface == "switch":
+            cmd = params["bridge_get_if_cmd"]
+            output = process.system_output(cmd, shell=True).decode()
+            host_interface = output.strip(':')
+        elif host_interface == "ovs0":
+            cmd = "ovs-vsctl list-ports %s" % host_interface
+            output = process.system_output(cmd).decode()
+            host_interface = output
+
+    params["start_vm"] = "yes"
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    device_mac = vm.virtnet[0].mac
+    iface = vm.get_ifname()
+    # TBD, will support windows later
+    process.system_output(params["nm_stop_cmd"])
+    cmd = "ifconfig %s mtu 4000" % host_interface
```

Comment:
Name `check_guest_mtu_cmd` to `set_mtu_cmd` and equals to `"ifconfig %s mtu 4000"` and here directly `process.system_output(set_mtu_cmd % host_interface)`?